src/main/resources/application-dev.yml

```
@@ -20,7 +20,19 @@ spring:
-      org.hibernate.SQL: DEBUG
-      org.hibernate.type.descriptor.sql: TRACE
\ No newline at end of file
+        org:
+            springframework:
+            web: DEBUG
+            security: DEBUG
+  security:
+    jwt:
+      secret: ${JWT_SECRET:secret}
+      access-token-expiration: 3600
+      refresh-token-expiration: 604800
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)